src/test/javascript/spec/component/overview/course-conversations/layout/conversation-messages/conversation-messages.component.spec.ts

```diff
@@ -268,6 +270,23 @@ examples.forEach((activeConversation) => {
             expect(component.groupedPosts[2].posts).toHaveLength(1);
         });
 
+        it('should fetch forwarded messages correctly', () => {
+            const mockForwardedMessages: ForwardedMessage[] = [
+                { id: 101, sourceId: 10, sourceType: 'POST' } as unknown as ForwardedMessage,
```

Comment:
Does this not work without the double cast?

Reply:
I left it this way because I needed to add more valid fields, and otherwise it would have made the code even longer.